cmd/create.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright © 2014-2020 Nimble
```

Comment:
```suggestion
Copyright © 2014-2021 Nimble
```

Reply:
Fixed in 4b74b60b785dc6e389b0ee1f9ae891de84fcbad6

Reply:
Same as the comment below, instead follow the updated Git template.